In [5]:
##Question 1
Marks=int(input("Enter student percentage marks"))
if Marks>90:
    print("Student Got Grade A")
elif Marks>80:
    print("Student Got Grade B")
elif Marks>60:
    print("Student Got Grade C")
else:
    print("Student Got Grade D")

Enter student percentage marks 68


Student Got Grade C


In [12]:
#Question 2
Price=int(input("Enter the cost of Bike"))
if Price<=0:
          print("Enter valid price")
elif Price>100000:
          print("Your road tax is: ",Price*0.15)
elif Price>50000:
          print("Your road tax is: ",Price*0.10)
else:
          print("Your road tax is: ",Price*0.05)

          
          

Enter the cost of Bike -5555


Enter valid price


In [19]:
#Question 3
city=int(input("Enter the number of city to got its Monument: 1. Delhi, 2. Agra, 3. Jaipur"))
if city==1:
    print("Red Fort")
elif city==2:
    print("Taj Mahal")
elif city==3:
    print("Jal Mahal")
else:
    print("Please enter valid entry from 1,2,3")

Enter the number of city to got its Monument: 1. Delhi, 2. Agra, 3. Jaipur 3


Jal Mahal


In [24]:
#Question-4
num = int(input("Enter number"))
count=0
while num>=10 :
    count=count+1
    num=num//3
print('number can be divide by 3, ',count,' times')

Enter number 675


number can be divide by 3,  4  times


#Question-5
We use while loop to run a block of code untill a certain condition met. It check the condition, and if it true it run the code inside. and it check again untill the condition is true.
For example we can see question no 4 of this assignment. It runs the block code untill the num value is greater than(as per given condition).
Lets evaluate it:
We given input of 675.
it checks condition 675>10, conndition satisfied, block code runs.
675//3=225
now condition check again 225>10, block code will run again
225//3=75
now 75>10
75//3=25
25>10
25//3=8
And now condition failed because 8 is less than 10. So while loostops here.

In [3]:
#Question 6
a=15
b=1
c=12
d=1
while a>b:
    while b<5:
        print('@'*b)
        b=b+1
    while b<10:
        print('$'*c)
        c=c-2
        b=b+1
    while b<=15:
        print('&'*d)
        d=d+1
        b=b+1


@
@@
@@@
@@@@
$$$$$$$$$$$$
$$$$$$$$$$
$$$$$$$$
$$$$$$
$$$$
&
&&
&&&
&&&&
&&&&&
&&&&&&


In [4]:
#Question-7
a=10
while a>0:
    print(a)
    a-=1

10
9
8
7
6
5
4
3
2
1


#Question 8 & 7 are same